cloudant-client/src/test/java/com/cloudant/tests/HttpTest.java

```diff
@@ -345,6 +345,91 @@ public void cookieRenewal() throws Exception {
     }
 
     /**
+     * This test check that the cookie is renewed if the server presents a Set-Cookie header
+     * after the cookie authentication.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void iamCookieRenewal() throws Exception {
+        final String hello = "{\"hello\":\"world\"}\r\n";
+        final String authSession = "AuthSession=";
```

Comment:
I believe the IAM session is going to have a different name, should be `IAMSession=...`